Formula/libsql-server.rb

```diff
@@ -0,0 +1,30 @@
+class LibsqlServer < Formula
+  desc "Server-mode binary for libSQL (sqld), an Open Contribution fork of SQLite"
+  homepage "https://libsql.org"
+  # pull from git tag to get submodules
+  url "https://github.com/libsql/sqld.git",
+      tag:      "v0.8.0",
+      revision: "4de673d04214a28188a45eefb60d2b2cc403859e"
```

Comment:
```suggestion
  # pull from git tag to get submodules
  url "https://github.com/libsql/sqld/archive/refs/tags/v0.8.0.tar.gz"
  sha256 "0ea95ede3c8f76819d18e6c536c154c445460d10fa8646b8dad1bb91831a9457"
```

Reply:
Thanks Rui! For some reason the tarball build is failing =( Not sure if github is failing to include something in the image.

For posterity, it complains that:
```
Caused by:
  process didn't exit successfully: `/private/tmp/libsql-server-20230414-77139-w8tw5t/sqld-0.8.0/target/release/build/sqld-03e2c2f85f5ab9fc/build-script-build` (exit status: 1)
  --- stderr
  Error: could not find repository from '/private/tmp/libsql-server-20230414-77139-w8tw5t/sqld-0.8.0/sqld'; class=Repository (6); code=NotFound (-3)
warning: build failed, waiting for other jobs to finish...
error: failed to compile `sqld v0.8.0 (/private/tmp/libsql-server-20230414-77139-w8tw5t/sqld-0.8.0/sqld)`, intermediate artifacts can be found at `/private/tmp/libsql-server-20230414-77139-w8tw5t/sqld-0.8.0/target`
```

Nothing comes to mind right now, but let me take a deeper look at this and I'll get back here soon.

Thanks again for the prompt review!

Reply:
Ok. There is an issue upstream that prevents building from source.
For reference: https://github.com/libsql/sqld/pull/334

Trying to fix that, and release 0.8.1, then we can use that for homebrew. Thanks!